pkg/azuredisk/controllerserver.go

```diff
@@ -249,6 +249,12 @@ func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest)
 
 	klog.V(2).Infof("deleting azure disk(%s)", diskURI)
 	if isManagedDisk(diskURI) {
+		_, err := getResourceGroupFromDiskURI(diskURI)
```

Comment:
it's better write like this:
```
if _, err := getResourceGroupFromDiskURI(diskURI); err != nil {
			....
}
```

And for the diskURI validation, can we use this:
https://github.com/kubernetes/kubernetes/blob/4d380ee279aa207b0fd46f0464aba09a366818d9/pkg/apis/core/validation/validation.go#L1329

unmanged disk validation:
https://github.com/kubernetes/kubernetes/blob/4d380ee279aa207b0fd46f0464aba09a366818d9/pkg/apis/core/validation/validation.go#L1333

Reply:
Sure